src/workflow/tests/database_models_test.py

```diff
@@ -0,0 +1,246 @@
+from workflow.database.report.models import (
+    DataRun,
+    InstrumentManager,
+    DataRunManager,
+    WorkflowSummary,
+)
+
+import unittest.mock as mock
+import unittest
+
+
+class TestModelsInstrumentManager(unittest.TestCase):
```

Comment:
Can the tests be changed to inherit from `django.test.TestCase` to be consistent with the other tests